test/powershell/Modules/Microsoft.PowerShell.Management/FunctionProvider.Tests.ps1

```diff
@@ -59,4 +59,34 @@ Describe "Basic Function Provider Tests" -Tags "CI" {
             { Set-Item $nonExistingFunction -Value 123 -ErrorAction Stop } | ShouldBeErrorId "Argument,Microsoft.PowerShell.Commands.SetItemCommand"
         }
     }
+
+    Context "Validate Get-Item Cmdlet" {
+        It "Gets existing functions by name" {
+            $getItemResult = Get-Item $existingFunction
+            $getItemResult.Name | Should -Be $existingFunction
+            $getItemResult.Options | Should -Be "None"
+            $getItemResult.ScriptBlock | Should -Be $functionValue
+        }
+
+        It "Matches regex with stars to the function names" {
+            $getItemResult = Get-Item "ex*on"
+            $getItemResult.Name | Should -Be $existingFunction
+
+            # Stars representing empty string.
+            $getItemResult = Get-Item "*existingFunction*"
+            $getItemResult.Name | Should -Be $existingFunction
+
+            # Finds 2 functions that match the regex.
+            Set-Item $nonExistingFunction -Value $functionValue
+            $getItemResults =  Get-Item "*Function"
+            $getItemResults.Count | Should -BeGreaterThan 1
+        }
+    }
+
+    Context "Validate Remove-Item Cmdlet" {
+        It "Removes function" {
+            Remove-Item $existingFunction
+            $existingFunction | Should -Not -Exist
```

Comment:
This is incorrect.. $existingFunction is a string so this condition will always be true. You should be checking by using Get-Item